source/collaborate/channel-types.rst

```diff
@@ -29,13 +29,17 @@ Channel members can choose to leave private channels at any time.
 Direct messages
 ---------------
 
-Direct messages are channels for conversations between two people. You can also start a direct message with people on other teams when enabled by the System Admin. Only members of the conversation can see direct messages and channel heading information, including last active status of the other user, when enabled.
+Direct messages are channels for conversations between 2 people. You can start a direct message with people on other teams `when enabled by the system admin </configure/site-configuration-settings.html#users-restrictdirectmessage>`__. Only members of the conversation can see direct messages and channel heading information, including the last active status of the other user. 
+
+Direct messages always display a new message badge. Direct messages always trigger a notification, unless you mute the channel, or you disable notifications altogether. See the `notification documentation </preferences/manage-your-notifications.html>`__ for details on customizing notifications based on your preferences.
 
 Group messages
 --------------
 
-Group messages are channels for conversations among three or more people and up to seven people. You can also start a group message with people on other teams when enabled by the System Admin. Only members of the conversation can see group messages.
+Group messages are channels for conversations among 3 or more people and up to 7 people. You can also start a group message with people on other teams when `enabled by the system admin </configure/site-configuration-settings.html#users-restrictdirectmessage>`__. Only members of the conversation can see group messages. Group messages always display a new message badge.
```

Comment:
Group messages always display a new message badge... after v9.1, not before. Not sure if you want to make that distinction or not.